src/nxt_conf_validation.c

```diff
@@ -2798,6 +2831,35 @@ nxt_conf_vldt_target(nxt_conf_validation_t *vldt, nxt_str_t *name,
 }
 
 
+#if (NXT_HAVE_CGROUP)
+
+static nxt_int_t
+nxt_conf_vldt_cgroup_path(nxt_conf_validation_t *vldt, nxt_conf_value_t *value,
+    void *data)
+{
+    char       path[NXT_MAX_PATH_LEN];
+    nxt_str_t  cgpath;
+
+    nxt_conf_get_string(value, &cgpath);
+    if (cgpath.length >= NXT_MAX_PATH_LEN - strlen(NXT_CGROUP_ROOT) - 1) {
```

Comment:
This check doesn't cover `"/cgroup.procs"`, does it?

Reply:
It doesn't cover the relative case either, AFAICS.

Reply:
If we don't test here, that's fine, but we'll need to know that we need to truncate later.  So, my previous simplification proposal would be invalid, and we'd need to add `stpecpy()` at some point.  We can do that after the next release.

Reply:
No, but then we have further checks. As mentioned, that's just a quick sanity check for the absolute path case.